core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -269,6 +269,27 @@ private[spark] object Utils extends Logging {
     file.setExecutable(true, true)
   }
 
+  /**
+   * Move data to trash if 'spark.sql.truncate.trash.enabled' is true
+   */
+  def moveToTrashIfEnabled(
+      fs: FileSystem,
+      partitionPath: Path,
+      isTrashEnabled: Boolean,
+      hadoopConf: Configuration): Boolean = {
```

Comment:
This method name and doc is actually not accurate. It will delete the path, not just move the data to trash. Actually it is short and can easily get it by looking into the code. But it is nicer if we give it correct method name and correct doc.

Reply:
I have updated the description, please suggest if it is okay now.